.releaserc

```diff
@@ -1,4 +1,12 @@
 {
+  "branches": [
+    "+([0-9])?(.{+([0-9]),x}).x",
+    "main",
+    "next",
+    "next-major",
+    { name: "beta", prerelease: true },
+    { name: "alpha", prerelease: true },
+  ],
```

Comment:
I see this is a modification of [the default](https://semantic-release.gitbook.io/semantic-release/usage/configuration#branches) to use `main` instead of `master`. It seems a little weird, though, because we only release from `main`. Should we shorten this list?

Reply:
Since we need to update workflows anyway to release from other branches, I've shortened the list to `main`.